bubblesub/api/gui.py

```diff
@@ -28,6 +29,15 @@
 from bubblesub.ui.util import SUBS_FILE_FILTER, async_dialog_exec, save_dialog
 
 
+class ViewState(enum.IntEnum):
+    """The GUI views."""
+
+    Full = 0
+    Audio = 1
+    Video = 2
+    Subs = 3
```

Comment:
I'd use `StringEnum` to simplify the code the way I mentioned earlier

Maybe let's call it ViewLayout?